src/Raven.Client/Documents/Session/Loaders/IncludeBuilder.cs

```diff
@@ -525,6 +538,27 @@ private void IncludeCountersWithAlias(Expression<Func<T, string>> path, string[]
             IncludeCounters(path.ToPropertyPath(), names);
         }
 
+        private void IncludeRevisionsBefore(DateTime? dateTime)
+        {
+            if (dateTime == null)
+                return;
+            
+            if(dateTime == default(DateTime))
+                throw new ArgumentNullException(nameof(dateTime));
+    
+            RevisionsToIncludeByDateTime ??= new DateTime();
```

Comment:
Why?

Reply:
In the JINT side it will always initialize date time for the first time